### DISCUSSION TITLE
Updating data on two axes with different timeframes causes out of sync issues

### DISCUSSION THREAD

**Discussion by xabaddonx1** [highfestiva/finplot#297] [Q&A]
I have 2 different axes displaying data with 2 different timeframes (5 min and 30 min). It works fine if the data is static, but when I start updating the plot data, it can cause the timeframes to become out of sync (for the same X point, the time on the 30 min chart gets shifted back 1-2 hours randomly). I am using unix timestamps as the X coordinate, and the timestamp of the first element in both sets of data is the same. If I pan the graph around it fixes it, but as soon as the plot updates again it happens again. Am I doing something wrong?

**Comment by highfestiva** [comment]
Probably a bug caused by one being out of sync with the other in terms of data series length 5/6ths of the time. If you just update both every half hour I suspect it might work. I would need a minimal example (that updates at runtime, not every 5 minutes) to ameliorate.
